### DISCUSSION TITLE
How to monitor the scroll position in modal after opening with up-layer="new"

### DISCUSSION THREAD

**Discussion by hansipete** [unpoly/unpoly#527] [Q&A]
I'm trying to mimic the "scroll back to overview" interaction that can be seen on https://heavygermanshit.de/.
I can't figure out how to keep my global (window) scroll event listener firing after opening the modal.
This is how my links look:
```
<a  href="/page" up-layer="new">
  Open Page
 </a>
 ```
 Hope somebody can help out. Thank you!

**Comment by triskweline** [comment]
While an overlay is open, the scrolling element is the overlay's viewport element. E.g. `<up-modal-viewport>`.

**Reply by hansipete** [reply]
Thank you – that makes sense!

Is there any way to bubble (is this the right term?) the scroll event further up them DOM? 

I have a `scroll` event listener on `document` that does not get fired when the overlay is opened. And this is what I think I would need to automatically `dismiss()` the overlay when scrolling far enough.

**Reply by hansipete** [reply]
I solved it by redesigning my master layout. I do have a .overlay container now that I fill with the content when needed. The links to details pages look like this:
```
<a href="details/a" up-target="[up-overlay]" up-transition="overlay">
```

in my master there is container `<div up-overlay>` that this content is rendered into.
